### ISSUE TITLE
If the QE uses a Compendium Actor, check for a previously extracted version of the Actor

### ISSUE THREAD

**Issue by spetzel2020** [spetzel2020/quick-encounters#66]
Currently when QE calls `getActor()` it does an `importFromCollection()` which always creates a new Actor, even if that Actor has already been imported.

Instead, check and don't re-import the Actor if it already exists (hopefully with some basic checking that it's the same as the one in the Compendium, rather than just matching on name - perhaps by setting a `fromCompendiumId` flag when first extracted)

**Comment by sneat** [comment]
There should be a `sourceID` flag set by core when you import from a compendium and it should then have the `uuid` reference of the compendium entry.

`actor.getFlag('core', 'sourceId')`

For example:

```
game.actors.getName('Shadow').getFlag('core', 'sourceId')
> 'Compendium.dnd5e.monsters.QGcQYZbVl4bWzi4E'
```

**Comment by spetzel2020** [comment]
Fixed and released in 0.9.10 (see Issue #81 )
